org.kde.iconexplorer.yml

```diff
@@ -0,0 +1,94 @@
+app-id: org.kde.iconexplorer
+runtime: org.kde.Platform
+runtime-version: 6.8
+sdk: org.kde.Sdk
+command: iconexplorer
+copy-icon: true
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.freedesktop.FileManager1
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/libexec
+  - /mkspecs
+  - /etc/xdg/*categories
+  - /share/qlogging-categories5
```

Comment:
I think you should keep those two. The logging category are used for logging. The other one I don't know but likely used for the desktops entry.